extensions/rocksdb-repos/DatabaseContentRepository.cpp

```diff
@@ -38,8 +39,8 @@ bool DatabaseContentRepository::initialize(const std::shared_ptr<minifi::Configu
   }
   rocksdb::Options options;
   options.create_if_missing = true;
-  options.use_direct_io_for_flush_and_compaction = true;
-  options.use_direct_reads = true;
+//  options.use_direct_io_for_flush_and_compaction = true;
+//  options.use_direct_reads = true;
```

Comment:
I didn't mean to commit this part, but I will note: on my machine this was causing build failure, even on master, so we probably will need to look at this separately.